flow/utils/leaderboard/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM continuumio/miniconda3:4.5.4
+MAINTAINER Fangyu Wu (fangyuwu@berkeley.edu)
+
+# System
+RUN apt-get update && \
+	apt-get -y upgrade && \
+	apt-get install -y \
+    vim \
+    apt-utils
+
+# Flow
+RUN cd ~ && \
+	git clone https://github.com/flow-project/flow.git && \
+    cd flow && \
+    git checkout leaderboard && \
```

Comment:
If we're merging this into master, is the `git checkout leaderboard` going to be necessary?

Reply:
My plan is to keep the leaderboard branch as sort of devel branch and only push stable, tested codes to master once in a while. I just realized that I failed all the checks, again... Let me fix this!

Reply:
switch to master